.eslintrc.js

```diff
@@ -101,6 +101,16 @@ module.exports = {
     'simple-import-sort/imports': 'warn',
 
     'import/newline-after-import': ['warn', { 'count': 2 }],
-    'import/no-anonymous-default-export': ['warn', { 'allowObject': true }]
-  }
+    'import/no-anonymous-default-export': ['warn', { 'allowObject': true }],
+
+    'no-debugger': 'warn'
+  },
+  'overrides': [
+    {
+      'files': 'src/**/*.js',
```

Comment:
Only enabling `no-console` for the contents of the `src` directory since there are plenty of valid uses for console logs in integration tests, configuration, scripts, etc.